online-dashboard-app/src/Components/User/JobsTable.jsx

```diff
@@ -1,60 +1,115 @@
-            <th className="p-3 text-left">Qualification</th>
-            <th className="p-3 text-left">Apply Link</th>
-          </tr>
-        </thead>
-        <tbody>
-          {currentJobs.map((job) => (
-            <tr key={job.id} className="border-t hover:bg-gray-50 transition">
-              <td className="p-3">{job.company_name}</td>
-              <td className="p-3">{job.role}</td>
-              <td className="p-3">{job.batch}</td>
-              <td className="p-3">{job.qualification}</td>
-              <td className="p-3">
-
-                <button
-                  className="bg-blue-500 text-white py-2 px-4 rounded-md hover:bg-blue-600"
-                >
-                  <a
-                    href={job.apply_link}
-                    target="_blank"
-                    rel="noopener noreferrer"
-                    className="text-black-600 hover:underline red"
-                  >
-                    Apply
-                  </a>
-                </button>
-              </td>
+    <div className={`bg-white shadow-md rounded-lg ${className}`}>
+      {/* Table for larger screens */}
+      <div className="hidden md:block">
+        <table className="w-full table-auto">
+          <thead>
+            <tr className="bg-indigo-100 text-indigo-800 font-semibold">
+              <th className="p-3 text-left">Company Name</th>
+              <th className="p-3 text-left">Role</th>
+              <th className="p-3 text-left">Batch</th>
+              <th className="p-3 text-left">Qualification</th>
+              <th className="p-3 text-left">Apply Link</th>
             </tr>
+          </thead>
+          <tbody>
+            {currentJobs.map((job) => (
+              <tr key={job.id} className="border-t hover:bg-gray-50 transition">
+                <td className="p-3">{job.company_name}</td>
+                <td className="p-3">{job.role}</td>
+                <td className="p-3">{job.batch.split(",").join(", ")}</td>
+                <td className="p-3">{job.qualification.split(",").join(", ")}</td>
+                <td className="p-3">
+                  <button className="bg-blue-500 text-white py-2 px-4 rounded-md hover:bg-blue-600">
+                    <a
+                      href={job.apply_link}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      className="text-black-600 hover:underline red"
+                    >
+                      Apply
+                    </a>
+                  </button>
+                </td>
+              </tr>
+            ))}
+          </tbody>
+        </table>
+      </div>
+
+      {/* Card view for smaller screens */}
+      <div className="block md:hidden">
+        <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-6">
+          {currentJobs.map((job) => (
+            <div key={job.id} className="border border-gray-200 p-4 rounded-lg shadow-md hover:shadow-xl transition">

```

Comment:
This is good!
But we actually need to create a seperate componen for the below div and call the component here by passing the values